diff --git a/dist/index.cjs b/dist/index.cjs
index e552163df832aa02286da63d90b6a83dfd3fc392..4fb5258d4500e8ba0346ee5639655cc729342902 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -182,6 +182,9 @@ To hide this message set build.sourcemap to true, 'inline' or 'hidden'.`)}`
         const combinedSourceMap = sanitizeSourceMap(
           this.getCombinedSourcemap()
         );
+        combinedSourceMap.sources.forEach((code, index) => {
+          combinedSourceMap.sources[index] = code.replace(/\?.*$/, '');
+        })
         const code = instrumenter.instrumentSync(
           srcCode,
           filename,
diff --git a/dist/index.mjs b/dist/index.mjs
index 34e886906b6527286eae6978ce6e73bde396470f..6d224c417b00523d5ab7ddc7a4022f389e8cfd4b 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -162,6 +162,9 @@ To hide this message set build.sourcemap to true, 'inline' or 'hidden'.`)}`
         const combinedSourceMap = sanitizeSourceMap(
           this.getCombinedSourcemap()
         );
+        combinedSourceMap.sources.forEach((code, index) => {
+          combinedSourceMap.sources[index] = code.replace(/\?.*$/, '');
+        })
         const code = instrumenter.instrumentSync(
           srcCode,
           filename,
